process/modules/common.py

```diff
@@ -40,6 +54,11 @@ def clipAreasByAreas(geometryToClip: gpd.GeoDataFrame, mask: gpd.GeoDataFrame, g
 
     # Adding not clipped objects
     retval = gpd.GeoDataFrame(pd.concat([retval, not_clipped], ignore_index=True))
+    retval["geometry"] = retval.make_valid()
+    retval["geometry"] = retval.normalize()
+    retval.drop_duplicates()
+    retval["geometry"] = retval.buffer(0.1)
+    retval["geometry"] = retval.buffer(-0.1)
```

Comment:
There's probably a reason for doing this, even though on the surface it looks like immediately undoing what was just done. This could be explained with a comment.

Reply:
Comment added